src/components/layout/page/page.tsx

```diff
@@ -18,16 +25,26 @@ export const Page: React.FC<PageProps> = ({
   children,
   pad = false,
   valign,
+  backgroundImageUrl,
+  backgroundOverlayOpacity = 0.5,
   ...props
-}) => (
-  <div
-    className={clsx("mykn-page", {
-      [`mykn-page--valign-${valign}`]: valign,
-      "mykn-page--pad-h": pad === true || pad === "h",
-      "mykn-page--pad-v": pad === true || pad === "v",
-    })}
-    {...props}
-  >
-    {children}
-  </div>
-);
+}) => {
+  return (
+    <div
+      className={clsx("mykn-page", {
+        [`mykn-page--valign-${valign}`]: valign,
+        "mykn-page--pad-h": pad === true || pad === "h",
+        "mykn-page--pad-v": pad === true || pad === "v",
+      })}
+      {...props}
+    >
+      {backgroundImageUrl && (
+        <BackgroundImage
```

Comment:
I don't think BackgroundImage needs to be available as a separate component, but I like the implementation as prop on page. Maybe just not export in and but BackgroundImage in the same file of Page?